wallet/wallet.sol

```diff
@@ -337,7 +337,9 @@ contract Wallet is multisig, multiowned, daylimit {
     // and _data arguments). They still get the option of using them if they want, anyways.
     function execute(address _to, uint _value, bytes _data) external onlyowner returns (bytes32 _r) {
         // first, take the opportunity to check that we're under the daily limit.
-        if (underLimit(_value)) {
+        // we also must check that there is no data (this is not a contract invocation),
```

Comment:
thank you, i updated the diff to also check if the other address has code. 
I wonder if we still need to check for _data.length here?